nevergrad/functions/images/imagelosses.py

```diff
@@ -3,38 +3,104 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import cv2  # type: ignore
+import imquality.brisque as brisque  # type: ignore
```

Comment:
instead of adding type ignore here, please append to the list of ignored imports:
https://github.com/facebookresearch/nevergrad/blob/493b526ebe0c210cdc709184b95a35e66b73a3ec/mypy.ini#L4

Reply:
done.